.readthedocs.yml

```diff
@@ -0,0 +1,20 @@
+# .readthedocs.yml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+version: 2
+
+sphinx:
+  builder: html
+  configuration: docs/conf.py
+  fail_on_warning: true
+
+formats: all
+
+python:
+  version: 3.6
+  install:
+    - requirements: dev_requirements.txt
```

Comment:
Note that this does *not* contain the pip, since it is regarded as unsafe in a .txt. So maybe add `dev_requirements.in` as well, but after `dev_requirements.txt`?